### ISSUE TITLE
[Understanding Conformance] Link & text missing from end of "Understanding Accessibility-Supported Web Technology Uses" section

### ISSUE THREAD

**Issue by ashleemboyer** [w3c/wcag#3472]
On the [WCAG 2.2 Understanding Conformance](https://www.w3.org/WAI/WCAG22/Understanding/conformance) page, at the very end of the [Understanding Accessibility-Supported Web Technology Uses](https://www.w3.org/WAI/WCAG22/Understanding/conformance#documented-lists) section, there is missing text after the word "See":

![Screenshot of the quoted text with a red circle around the sentence with missing text.](https://github.com/w3c/wcag/assets/43934258/fe01ee80-fec9-41f2-be24-711e2ed7419d)

After searching this repo for the nearby text content, and based on the context of that text, it looks like there is text missing from a link. I believe this is the source code for the page:

https://github.com/w3c/wcag/blob/main/understanding/conformance.html#L802-L803

The `href` of that link currently references an ID `#accessibility-support-documenting` but there is no element on the page with that ID. I searched for that text in this repository. Based on what appears to be [the `2.0` source code for this page](https://github.com/w3c/wcag/blob/main/wcag20/sources/understanding/conformance.xml#L278) and the existence of [a file with a very similar name (`documenting-accessibility-support`)](https://github.com/w3c/wcag/blob/main/understanding/documenting-accessibility-support.html), I believe this link is supposed to point to a different page.

To summarize, I believe [the sentence in question](https://github.com/w3c/wcag/blob/main/understanding/conformance.html#L802-L803) needs to be updated to something like the following:

```
See <a href="https://www.w3.org/WAI/WCAG22/Understanding/documenting-accessibility-support">Documenting Accessibility Support for Uses of a Web Technology</a>.
```

Further testing shows that this issue also exists for the 2.1 and 2.0 versions of this page:

- https://www.w3.org/WAI/WCAG21/Understanding/conformance#documented-lists
- https://www.w3.org/WAI/WCAG20/Understanding/conformance#documented-lists

**Comment by iadawn** [comment]
Fixed for 2.0 and 2.1
